### ISSUE TITLE
Exercise 2.3.13(d)

### ISSUE THREAD

**Issue by awgu** [UlisseMini/understanding-analysis-solutions#1]
- Fix `\epsilon > 0`. Since `\lim_{n \rightarrow \infty} a_{mn} = b_m` for all `m \in \mathbb{N}`, there exists `N_m \in \mathbb{N}` such that for all `n \geq N_m`, `|a_{mn} - b_m| < \epsilon`.
- Notably, the choice of lower bound `N_m` depends on `m`; i.e. it may be different for different `m`. Because of this, I am not following how we can choose a sufficiently large lower bound `N` when proving the limit `\lim_{m \rightarrow \infty} b_m = a`.
- You seem to be implicitly taking a maximum over the sequence `N_1, N_2, \dots, N_m, \dots`, but that requires showing that the sequence is bounded (e.g. by showing that it converges). However, I am not sure how to do that.

Let me know if you want me to clarify my concern in any way.

**Comment by UlisseMini** [comment]
Oops you're totally right, I ignored the dependence on `m`. Working on fixing that now

**Comment by UlisseMini** [comment]
@awgu See if the new explanation makes sense

**Comment by awgu** [comment]
Sorry, at the moment it seems to me that the issue remains. What exactly is your final `N` you are choosing to show that `|b_m - a| < \epsilon` for all `m \geq N`? Again, (using your new notation) it seems like it must be `\max\{N, N_1, \dots, N_m, \dots\}`, where `N` is the one fixed for `|a_{mn} - a| < \epsilon / 2` and each `N_1, \dots, N_m, \dots` is for `|b_m - a_{mn}| < \epsilon / 2` for each `m`.

My concern is that each `N_m` may be greater than `N`, and we have not bounded the set of all `N_m`. Hence, the `max` over `N, N_1, \dots, N_m, \dots` may not be well-defined. Given your setup, we need that `max` so that `|b_m - a_{mn}| < \epsilon / 2` or else even with the triangle inequality, we cannot conclude that `|b_m - a| < \epsilon / 2 + \epsilon / 2 = \epsilon`.

Let me know if I am missing something.

**Comment by UlisseMini** [comment]
It isn't the maximum, I choose `N` so that `m,n > N` implies `|a_{mn} - a| < epsilon/2` then I show that with this same `N`, `m > N` implies `|b_m - a| < epsilon`.

I think what I did is unclear since `epsilon/2` is arbitrary, I've rewritten it to account for that. We're just taking advantage of `n` being a variable *we introduced* so we can set it to whatever we want (including making it dependent on `m`) as we're in the "verify that `m > N` works" phase.
